packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -282,12 +282,16 @@ const typeStringAndroid = async (
 };
 
 // Calculates middle x,y and clicks that position
-const clickMiddleOfElement = async ( driver, element ) => {
+const clickMiddleOfElement = async ( driver, element, isParagraph = false ) => {
 	const location = await element.getLocation();
 	const size = await element.getSize();
-
 	const action = await new wd.TouchAction( driver );
-	action.press( { x: location.x + size.width / 2, y: location.y } );
+	// Condition to make it work on iOS Paragraph blocks on both environments, the default calculation somehow doesn't work for this combination but works for others.
+	if ( isParagraph === true && ! isAndroid() ) {
+		action.press( { x: size.width / 2, y: location.y } );
```

Comment:
Thanks for questioning this. I just checked again, and it does work without needing the changes! I will revert the change as well as the alternative assertion that strips away white spaces and keep the original code in place. 

In my earlier tests, the coordinates did not work locally, and after making this change it started working but then I made additional changes along the way so it could be that I messed something up earlier.

Reply:
I see this was solved in https://github.com/WordPress/gutenberg/pull/40404/commits/eaa7e2d9acc7b25141a7ec8dcf56da2791912e60 so I'm going to mark the conversation as resolved 🎊 .

Reply:
Oops sorry forgot to update that this has been updated. I also reverted some of the changes I made that were needed by the coordinate change, so there are fewer changes now 😄